src/std/env.ab

```diff
@@ -15,59 +16,71 @@ pub fun get_env_var(var: Text, file: Text = ".env"): Text {
     return ""
 }
 
+/// Load the env file in the environment
 pub fun load_env_file(file: Text = ".env"): Null {
     unsafe $export "\$(xargs < {file})" > /dev/null$
 }
 
+/// ?
 pub fun shell_isset(name: Text): Bool {
     $[[ ! -z \$\{!{nameof name}+z} ]]$ failed {
         return false
     }
     return true
 }
 
+/// Set a constant
 pub fun shell_constant_set(name: Text, val: Text): Null {
     $readonly \${nameof name}="\${nameof val}" 2> /dev/null$?
 }
 
+/// Get a constant
 pub fun shell_constant_get(name: Text): Text {
     return $echo \$\{!{nameof name}}$?
 }
 
+/// Set a variable?
```

Comment:
@CymDeveloppement

Reply:
sorry, yes this function set a shell variable